SuperdenseCoding/Tasks.qs

```diff
@@ -3,11 +3,16 @@
 
 namespace Quantum.Kata.SuperdenseCoding {
     
-    open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Canon;
-    
+    open Microsoft.Quantum.Measurement;
     
+    /// # Summary
+    /// Data type that represents the message that is transmitted 
+    /// as part of the superdense coding protocol. 
+    /// It includes two classical bits (Bit1 and Bit2) of type Bool.   
+    newtype Message = (Bit1 : Bool, Bit2 : Bool);    
```

Comment:
I believe the compiler doesn't mind, but for me type `Message` just reads as logging function `Message` and is confusing... Could we call it `ProtocolMessage` instead?

Reply:
Sure.